src/InlineBalance.jsx

```diff
@@ -15,93 +16,105 @@ class InlineBalance extends ReactiveComponent {
 
 	precise () {
 		let s = this.state;
+
 		return s.precise === null ? this.props.precise : s.precise;
 	}
 
 	togglePrecise () {
 		let s = this.state;
+
 		s.precise = !s.precise;
 		this.setState(s);
 	}
 
 	denom () {
 		let s = this.state;
+
 		return s.denom === null ? usableDenoms.indexOf(this.props.units) : s.denom;
 	}
 
 	cycleDenom () {
 		let denom = (this.denom() + 2) % (usableDenoms.length + 1) - 1;
-		this.setState({denom});
+
+		this.setState({ denom });
 	}
 
 	readyRender () {
 		let v = new BigNumber(this.state.value || 0);
 		let isNeg = v.lt(0);
 		let s;
 		let sd = this.denom();
```

Comment:
Idem here. When doing complicated stuff, try to put as much comments.

Maybe in other files too.